examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -174,6 +174,7 @@ grep -q SUCCESS run.daint.out
 status2=$?
 set -e
 wait
+python examples/standalone/benchmarks/collect_memory_usage_data.py . $githash
```

Comment:
I would move this to the end of the script. I think it would make sense to get the successful message _before_ getting a potential error message from extracting memory usage data.

Reply:
I could also imagine removing this here entirely and moving it into `.jenkins/actions/run_standalone.sh` before L94. The hardcoded path here will fail in case `run_on_daint.sh` is launched manually from this directory. Alternatively, you could simply add `$ROOT/` before the path here.

Reply:
good catch. I think moving it into `.jenkins/actions/run_standalone.sh` is the wrong approach:
- The timing json-file is also written in `run_on_daint`, so it seems inconsistent to write the memory not also here
- It is a bit tedious because then we have to only source a venv for this one line.
So I moved it to the end

Regarding the root dir:
This is not a problem. We have a few places with relative path such as 
https://github.com/VulcanClimateModeling/fv3core/blob/7dd3a015648a41094d842fc25f81915492743b4c/examples/standalone/benchmarks/run_on_daint.sh#L126
or even 
https://github.com/VulcanClimateModeling/fv3core/blob/7dd3a015648a41094d842fc25f81915492743b4c/examples/standalone/benchmarks/run_on_daint.sh#L132
but this is all fine because we cd to root in
https://github.com/VulcanClimateModeling/fv3core/blob/7dd3a015648a41094d842fc25f81915492743b4c/examples/standalone/benchmarks/run_on_daint.sh#L86
if you want the root to be here for readability we can do it - I don't think it is necessary though

Reply:
Agree.